specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -5398,6 +6231,57 @@
         }
       ],
       "x-ms-discriminator-value": "Custom"
+    },
+    "InferenceEndpointCreateRequest": {
+      "description": "The request to create an Inference Endpoint.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the endpoint.",
```

Comment:
needs to be regionally unique?

Reply:
there is a property for location. we can do a check based on location to see if the name already exists.